src/Service/Storage/Ef/DbGlobalContext.cs

```diff
@@ -87,9 +90,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.Entity<ApplicationEvent>(builder =>
         {
             builder.HasKey(x => x.Id);
```

Comment:
This adds backwards compatibility, so we don't need to migrate any data, it just maps it to the new CLR property.